frontend/resourceadm/components/ImportResourceModal/ImportResourceModal.tsx

```diff
@@ -52,124 +38,48 @@ export const ImportResourceModal = ({
   };
 
   /**
-   * Updates the value of the title. If the edit field is not open,
-   * then it updates the ID to the same as the title.
-   *
-   * @param val the title value typed
+   * Import the resource from Altinn 2, and navigate to about page on success
    */
-  const handleEditTitle = (val: string) => {
-    if (!editIdFieldOpen && bothFieldsHaveSameValue) {
-      setId(val.replace(/\s/g, '-'));
-    }
-    setTitle(val);
-  };
-
-  /**
-   * Handles the click of the edit button. If we click the edit button
-   * so that it closes the edit field, the id is set to the title.
-   *
-   * @param isOpened the value of the button when it is pressed
-   */
-  const handleClickEditButton = (isOpened: boolean, isSave: boolean) => {
-    setEditIdFieldOpen(isOpened);
-
-    if (isSave) {
-      setBothFieldsHaveSameValue(false);
-    } else {
-      if (!isOpened) {
-        setBothFieldsHaveSameValue(true);
-        // If we stop editing, set the ID to the title
-        if (title !== id) setId(title.replace(/\s/g, '-'));
-      }
-    }
-  };
-
-  const handleSelectService = (s: string) => {
-    setSelectedService(s);
-    handleEditTitle(s);
-  };
-
-  /**
-   * Display loading (todo), the service, or nothing based on the
-   * state of the selected environment
-   */
-  const displayService = () => {
-    // If environment loading, display loading, else do below
-    if (selectedEnv) {
-      return (
-        <div className={classes.dropdownWraper}>
-          <Select
-            // Replace below with the real service list from API
-            options={dummyServices.map((s) => ({ value: s.name, label: s.name }))}
-            onChange={handleSelectService}
-            value={selectedService}
-            label={t('resourceadm.dashboard_import_modal_select_service')}
-          />
-        </div>
-      );
-    }
-    return null;
-  };
-
-  /**
-   * Display loading (todo), the title and id, or nothing based on the
-   * state of the selected service
-   */
-  const displayTitleAndId = () => {
-    // If service loading, display loading, else do below
-    if (selectedService) {
-      return (
-        <>
-          <div className={classes.contentDivider} />
-          <ResourceNameAndId
-            isEditOpen={editIdFieldOpen}
-            title={title}
-            text={t('resourceadm.dashboard_import_modal_resource_name_and_id_text')}
-            id={id}
-            handleEditTitle={handleEditTitle}
-            handleIdInput={handleIDInput}
-            handleClickEditButton={(isSave: boolean) =>
-              handleClickEditButton(!editIdFieldOpen, isSave)
-            }
-            resourceIdExists={false} // TODO
-            bothFieldsHaveSameValue={bothFieldsHaveSameValue}
-          />
-        </>
-      );
-    }
-    return null;
-  };
-
-  // TODO when connected with API calls
   const handleImportResource = () => {
-    console.log('Importing... Coming soon');
+    importResourceFromAltinn2(
```

Comment:
should we add a mutation at the end to make it clear that this is a mutation? 🤔 
```suggestion
    importResourceFromAltinn2Mutation(
```